cmd/docker/main.go

```diff
@@ -156,11 +156,16 @@ func handleFlags() bool {
 func initLogging() error {
 	logcfg := viclog.NewLoggingConfig()
 	if *cli.debug || vchConfig.Diagnostics.DebugLevel > 0 {
-		logcfg.Level = log.DebugLevel
 		trace.Logger.Level = log.DebugLevel
```

Comment:
what is the default log level for trace and syslog is vhcConfig....DebugLevel==-1?

Reply:
stays at the default info level.  Not sure if we should change syslog level.